clang/lib/Sema/SemaBounds.cpp

```diff
@@ -435,33 +436,33 @@ namespace {
 }
 #endif
 
-// Convert all temporary bindings in an expression to uses of the values	
-// produced by a binding.   This should be done for bounds expressions that	
-// are used in runtime checks.  That way we don't try to recompute a	
-// temporary multiple times in an expression.	
-namespace {	
-  class PruneTemporaryHelper : public TreeTransform<PruneTemporaryHelper> {	
-    typedef TreeTransform<PruneTemporaryHelper> BaseTransform;	
-
-
-  public:	
-    PruneTemporaryHelper(Sema &SemaRef) :	
-      BaseTransform(SemaRef) { }	
-
-    ExprResult TransformCHKCBindTemporaryExpr(CHKCBindTemporaryExpr *E) {	
-      return new (SemaRef.Context) BoundsValueExpr(SourceLocation(), E);	
-    }	
-  };	
-
-  Expr *PruneTemporaryBindings(Sema &SemaRef, Expr *E, CheckedScopeSpecifier CSS) {	
-    // Account for checked scope information when transforming the expression.
-    Sema::CheckedScopeRAII CheckedScope(SemaRef, CSS);
-
-    Sema::ExprSubstitutionScope Scope(SemaRef); // suppress diagnostics	
-    ExprResult R = PruneTemporaryHelper(SemaRef).TransformExpr(E);	
-    assert(!R.isInvalid());	
-    return R.get();	
-  }	
+// Convert all temporary bindings in an expression to uses of the values	
```

Comment:
Nothing seems to have changed in this part of code?

Reply:
It seems that the line ending of these lines in the master version is different than the other lines (0d0a instead of 0a).
Should I keep the master version, or mine?

Reply:
Let's keep the master version.

Reply:
Reverted back to CRLF line ending.